index.ts

```diff
@@ -86,6 +88,32 @@ async function main() {
             }
             continue;
           }
+          // If devpool issue has the "Unavailable" label then close such issue
+          if ((devpoolIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.UNAVAILABLE))) {
+            if (devpoolIssue.state === "open") {
+              await octokit.rest.issues.update({
+                owner: DEVPOOL_OWNER_NAME,
+                repo: DEVPOOL_REPO_NAME,
+                issue_number: devpoolIssue.number,
+                state: "closed",
+              });
+              console.log(`Closed (bounty unavailable): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            } else {
+              console.log(`Already closed (bounty unavailable): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            }
+          } else {
+            if (devpoolIssue.state === "closed" && projectIssue.state === "open") {
+              await octokit.rest.issues.update({
+                owner: DEVPOOL_OWNER_NAME,
+                repo: DEVPOOL_REPO_NAME,
+                issue_number: devpoolIssue.number,
+                state: "open",
+              });
+              console.log(`Opened (bounty available): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            } else {
+              console.log(`Already opened (bounty available): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            }
+          }
```

Comment:
If we update issue's state to open/closed there there is no need for further updates and we can simply `continue` the iteration